modules/renter/contractor/contractor.go

```diff
@@ -248,7 +239,9 @@ func NewCustomContractor(cs consensusSet, w wallet, tp transactionPool, hdb host
 	}
 
 	// Mark contract utility.
-	c.managedMarkContractsUtility()
+	if err := c.managedMarkContractsUtility(); err != nil {
+		return nil, err
```

Comment:
I'm not sure that we want to return an error here, that means the contractor won't start up and that we won't be able to make one if there's an error setting the utility.

Reply:
Since we're now intending to persist the utility values, it's probably fine to just launch the threadedMaintenance function at startup and not worry about this at startup.

Reply:
On a related note, at startup we aren't calling tg.Stop() if the startup fails. I'm pretty sure that's a bug in the code, the contractor should be shut down fully if it encounters an error in the `New` function. Maybe we can make that a separate pull request.